From: Reiner Herrmann <reiner@reiner-h.de>
Date: Mon, 11 Apr 2016 20:07:03 +0200
Subject: documentation

Some documentation fixes

- add @dircategory / @direntry (found by Lintian)
- use makeinfo instead of obsolete texi2html
---
 docs/makefile        | 4 +---
 docs/usermanual.texi | 5 +++++
 2 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/docs/makefile b/docs/makefile
index 85ffab5..75ded9e 100644
--- a/docs/makefile
+++ b/docs/makefile
@@ -23,9 +23,7 @@ usermanual.pdf: usermanual.texi
 
 html/usermanual_1.html: usermanual.texi
 	for d in html; do [ -d $$d ] || mkdir -p $$d || exit 1; done
-	texi2html --expand tex --split chapter --no-iftex usermanual.texi
-	mv usermanual/*.html html
-	rmdir usermanual
+	makeinfo --html --no-split --no-headers -o usermanual.html usermanual.texi
 
 ship:
 	-rm -f *.{aux,cp,cps,dvi,fn,ky,log,pg,toc,tp,vr}
diff --git a/docs/usermanual.texi b/docs/usermanual.texi
index 4b21dcc..4a5690b 100644
--- a/docs/usermanual.texi
+++ b/docs/usermanual.texi
@@ -5,6 +5,11 @@
 @setchapternewpage odd
 @c %**end of header
 
+@dircategory Education
+@direntry
+* UCBLogo: (ucblogo).	Logo Programming Language
+@end direntry
+
 @iftex
 @vsize = 8.9 in
 @parskip = 7 pt
